mod/primitives/pkg/merkle/root_hasher_test.go

```diff
@@ -279,14 +279,14 @@ func requireGoHashTreeEquivalence(
 	// Ensure the lengths are the same
 	require.Equal(
 		t, len(expectedOutput), len(output),
-		fmt.Sprintf("Expected output length %d, got %d",
+		fmt.Errorf("Expected output length %d, got %d",
```

Comment:
@fridrik01 you can actually drop this. This is mostly CodeRabbit allucinating and asking for more context of test failures. However require already gives plenty of info here

Reply:
@fridrik01 what I mean is, you can drop the whole error message and just have `require.Equal(t, len(expectedOutput), len(output))`